TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+// This class makes a pair of color for major and minor color.
+{   class ColorPair
+    {   private Color majorColor;
+        public Color MajorColor
+        { get { return majorColor; } set { majorColor = value; } }
+        private Color minorColor;
+        public Color MinorColor
+        { get { return this.minorColor; } set { this.minorColor = value; } }
+        public override string ToString(){
+            Dictionary<string, int> whiteMap = new Dictionary<string, int>();
+            //for {minor color , painumber}
+            if (majorColor == Color.White){
+                whiteMap.Add("Blue",1);whiteMap.Add("Orange", 2);
+                whiteMap.Add("Green", 3);
+                whiteMap.Add("Brown", 4);whiteMap.Add("SlateGray", 5);}
+            if (majorColor == Color.Red){
+                whiteMap.Add("Blue", 6);whiteMap.Add("Orange", 7);
+                whiteMap.Add("Green", 8);whiteMap.Add("Brown", 9);
+                whiteMap.Add("SlateGray", 10);}
+            if (majorColor == Color.Black){

```

Comment:
hard coded, if a new color is introduced or a color is changed or a color is removed then you will have to rework this code

instead reuse the array and loop through